### ISSUE TITLE
Getting "Reference element is not the same element for which references were queried" in Terraform 0.12-formatted file.

### ISSUE THREAD

**Issue by manuelkiessling** [VladRassokhin/intellij-hcl#215]
### Prerequisites
 - [x] Ensure you have latest version of plugin installed
 - [x] Search for possible issue duplicates

### Installation details
 - [x] IDE version (`Help->About->Copy to Clipboard`)
```
IntelliJ IDEA 2019.2 EAP (Ultimate Edition)
Build #IU-192.4205.45, built on May 24, 2019
IntelliJ IDEA EAP User
Expiration date: June 23, 2019
Runtime version: 11.0.3+12-b248.2 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.5
GC: ParNew, ConcurrentMarkSweep
Memory: 1981M
Cores: 8
Registry: 
Non-Bundled Plugins: net.ashald.envfile, org.intellij.plugins.hcl, org.jetbrains.plugins.hocon, com.jetbrains.php, com.jetbrains.twig, de.espend.idea.php.annotation, org.intellij.scala, AWSCloudFormation, Pythonid, com.intellij.lang.puppet, fr.adrienbrault.idea.symfony2plugin, org.jetbrains.plugins.ruby, org.toml.lang, ru.adelf.idea.dotenv
```

 - [x] intellij-hcl plugin version (`Settings->Plugins`): `0.7.2`
 - [x] Terraform version (`terraform -v`): `v0.12.0`

### Terraform Configuration Files
See https://gist.github.com/manuelkiessling/4813a497533ef080a830bf0ba3659cbc#file-route53-tf

### Exception
```text
Reference element is not the same element for which references were queried

java.lang.RuntimeException: Element: org.intellij.plugins.hcl.psi.impl.HCLSelectExpressionImpl; reference: org.intellij.plugins.hil.psi.HCLElementLazyReference; reference element: org.intellij.plugins.hcl.psi.impl.HCLIdentifierImpl
```
See https://gist.github.com/manuelkiessling/4813a497533ef080a830bf0ba3659cbc#file-stacktrace-txt for full stacktrace.

### Expected Behavior
.tf file is correctly syntax highlighted in the IDE Editor with no issues reported, and no Exception.

### Actual Behavior
Lines https://gist.github.com/manuelkiessling/4813a497533ef080a830bf0ba3659cbc#file-route53-tf-L64-L65 are highlighted as errornous, and an IntelliJ exception occurs.

![grafik](https://user-images.githubusercontent.com/206592/58868464-b16a0600-86bc-11e9-8531-907b74cd7326.png)

However, Terraform itself works with the file flawlessly.

### Steps to Reproduce
1. Open file https://gist.github.com/manuelkiessling/4813a497533ef080a830bf0ba3659cbc#file-route53-tf in the Editor

**Comment by VladRassokhin** [comment]
While I cannot reproduce exception about references, parser error is still valid and would be fixed ASAP.

**Comment by VladRassokhin** [comment]
Fixed, will be available in 0.7.3
